src/caisy/entities.ts

```diff
@@ -0,0 +1,44 @@
+import { normalizeList } from "./utils";
+
+export const getEntitiesData = async (params: {
+  projectId: string,
+  query: string,
+}
+) => {
+  const { projectId, query } = params
+  const url = `https://cloud.caisy.io/api/v3/e/${projectId}/graphql`;
+
+  const response = await fetch(url, {
+    method: "POST",
+    headers: {
+      "Content-Type": "application/json",
+      "x-caisy-token": process.env.CMS_ACCESS_TOKEN,
+    },
+    body: JSON.stringify({
+      query,
+    }),
+  })
+
+  if (response.status === 401 || response.status === 403) {
```

Comment:
Same for this too, some `globalResponse` handler for `401`, `403` and non `200` responses.